modules/intro.md

```diff
@@ -0,0 +1,66 @@
+
+
+## D8 Contrib  Modules 
+
+As discussed in the introduction to this guide, the Drupal 8 Media ecosystem includes several independent but interoperable/pluggable Media modules.
+
+The modules are intended to be uses together as needed in order to provide as basic or advance a media solution as is needed for a particular Drupal project.
+
+
+###D8 Media Storage Modules
+
+* [**File Entity:**](modules/file_entity/intro.md) provides interfaces for storing and managing files, which extends core file entity e.g., allowing for file fields, types, display modes, etc. ([d.o](https://github.com/drupal-media/file_entity))
+ * *Latest Status:* TBA
+
+
+* [**Media Entity:**](modules/media_entity/intro.md) an alternate approach to media storage in Drupal, which doesn't directly extend core file entity ([d.o](https://www.drupal.org/project/media_entity))
+ * *Latest Status:* TBA
+
+
+* [**Flysystem:**](modules/flysystem/README.md) provides a bridge to use Flysystem as Drupal stream wrappers ([d.o](https://www.drupal.org/project/flysystem))
+ * *Latest Status:* TBA
+
+
+###D8 Media U/I Modules
+
+* [**Crop API:**](modules/crop/intro.md) an API for image cropping that's intended to be used with a related IP module such as [Image Widget Crop]( https://www.drupal.org/sandbox/woprrr/2571403) ([d.o](https://www.drupal.org/project/crop))
+ * *Latest Status:* TBA
+
+
+* [**DropzoneJS:**](modules/dropzonejs/intro.md) provides drag ’n’ drop file uploads with image previews ([d.o](https://www.drupal.org/project/dropzonejs))
+ * *Latest Status:* TBA
+
+
+* [**Embed API:**](modules/embed/intro.md) a framework module to embed things using the Drupal 8 WYSIWYG editor ([d.o](https://www.drupal.org/project/dropzonejs))
+ * *Latest Status:* TBA
+
+
+* [**Entity Browser:**](https://www.drupal.org/project/entity_browser) provide a generic browser that can be used for selecting file/media entities.
+ * *Latest Status:* TBA
+
+
+* [**Entity Embed:**](https://www.drupal.org/project/entity_embed) allows any entity to be embedded within a text area using a WYSIWYG editor.
+ * *Latest Status:* TBA
+
+
+* [**Image Widget Crop:**](modules/image_widget_crop/README.md) provides an interface for using the features of the [Crop API](https://www.drupal.org/project/crop) ([d.o](https://www.drupal.org/sandbox/woprrr/2571403))
+ * *Latest Status:* TBA
+
+
+* [**URL Embed:**](https://www.drupal.org/project/url_embed) allows embedding of URLs using the Drupal 8 WYSIWYG or link fields ([d.o](https://www.drupal.org/project/url_embed))
+ * *Latest Status:* TBA
+
+
+###D8 Media Display Modules
+
+* [**Fallback Formatter:**](https://www.drupal.org/project/fallback_formatter) a field formatter that attempt multiple formatters for displaying file and applies the first one that returns output.
```

Comment:
We should add Field formatter to this section (drupal.org/project/field_formatter).